src/AccessibilityInsights.SharedUx/Settings/ConfigurationConstants.cs

```diff
@@ -28,13 +28,15 @@ public partial class ConfigurationModel
         private const string DefaultHotKeySnap = "Shift + F8";
         private const string OldHotKeyRecord = "Shift + F6";
         private const int DefaultSelectionDelayMilliseconds = 100; // milliseconds
+        private const int DefaultColorContrastVersionThresholdSize = 10000; // pixels (height * width)
         internal const int MinimumSelectionDelayMilliseconds = 50;
 
         /// <summary>
         /// Strings to be used as key names in the dictionary--PLEASE RETAIN ALPHABETIC SORTING
         /// </summary>
         private const string keyAlwaysOnTop = "AlwaysOnTop";
         private const string keyAppVersion = "AppVersion";
+        private const string KeyColorContrastVersionThresholdSize = "ColorContrastVersionThresholdSize";
```

Comment:
The upper case 'K' here is intentional. PR #1402 makes this change for the other constants in this block